.github/ISSUE_TEMPLATE/unmaintained.yaml

```diff
@@ -0,0 +1,290 @@
+name: Unmaintained component
+description: Create an issue to track the deprecation of a component
+labels: ["help wanted", "unmaintained"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Please make sure to fill out the entire form below, providing as much context as you can in order to help us triage and track down your bug as quickly as possible.
```

Comment:
I don't have a great suggestion here, but maybe change the wording in the second half of this statement so it's not referring to this type of issue as a bug? (Same comment for the next line here as well)